lib/livebook_web/controllers/user_controller.ex

```diff
@@ -0,0 +1,14 @@
+defmodule LivebookWeb.UserController do
+  use LivebookWeb, :controller
+
+  def logout(conn, _params) do
```

Comment:
I would keep it in the auth controller/view, I think it fits there, and there's no need add microfiles unnecessarily :)

Reply:
We discussed this and the goal for now is to mirror LivebookWeb.UserPlug and LivebookWeb.AuthPlug. So I am fine with this but we can revisit later if necessary.